pegasus/sites.v3/code.org/views/census_map.haml

```diff
@@ -1,7 +1,15 @@
 %link{rel: "stylesheet", type: "text/css", href: "/css/census_map.css"}
 -max_height_percentage = 0.65 if max_height_percentage.nil?
 
-#gmap.full-width
```

Comment:
I think we have some JS that relies on the ID "gmap".

Reply:
There's still a #gmap id on the following line; I just wrapped the map and the footer in this #map id.  Do you think that will be an issue?

Reply:
Oh I see.  That should be okay.